source/Cosmos.System2/Graphics/Mode.cs

```diff
@@ -8,30 +8,36 @@ namespace Cosmos.System.Graphics
      */
     //public struct Mode : IEquatable<Mode>, IComparable<Mode>
```

Comment:
This is not directly related to this PR, but I think that it doesn't make sense to implement `IComparable<Mode>` here. Also, it doesn't seem to be used anywhere.